src/app/account/invitation/[inviteId]/signup/page.tsx

```diff
@@ -116,14 +129,16 @@ const SetupAccountForm: FC<InviteData> = ({ inviteId, email, orgName }) => {
                     />
 
                     <TextInput
+                        radius="sm"
```

Comment:
and set `flex` to the identical value here so they balance.

```suggestion
                    <TextInput
                        flex="1"
```